src/components/MainLayout.vue

```diff
@@ -33,7 +33,7 @@ export default {
         return {
           padding: '0px'
         }
-      }
+      } return undefined
```

Comment:
This is unnecessary

Reply:
Yea, i know.. But without it, i get this error
```
./src/components/MainLayout.vue
Module Error (from ./node_modules/eslint-loader/index.js):
error: Expected to return a value in "containerStyle" computed property (vue/return-in-computed-property) at src\components\MainLayout.vue:31:21:
  29 |       currentSelected: state => state.types.currentSelected
  30 |     }),
> 31 |     containerStyle: function () {
     |                     ^
  32 |       if (this.$route.path === '/') {
  33 |         return {
  34 |           padding: '0px'
```
I don't know exactly why this code worked without any issues before... It could be that they changed some things around in the newer version of vue (2.5.22 vs. 2.6.12) which got updated in package.json.

I mean, yeah.. It is basically an unnecessary return, but i really just wanted to assure proper functionality.

Here's a screenshot of what i'm talking about.. Note the fresh compile error exactly where i removed that unnecessary return?
![image](https://user-images.githubusercontent.com/1573345/100430026-846e9780-3096-11eb-84c0-071ac5d528d6.png)

Reply:
Okaaay i thiiiink i might have figured it out, but it needs testing.

I overlooked the fact that it isn't in fact an error with vue.js directly, rather it's an issue with eslint. Since it's a dev-dependency, it might work in production. Since i did not test production, that issue might have slipped past me.

Reply:
I see, seems like that's something new. And Googling about _some_ are saying that the computed property won't be recomputed if it returns undefined (though that seems unlikely)

I think returning an empty object is better in this case then
```suggestion
      } 
      return {}
```

Reply:
Nope, i figured out what happend. It's literally only just the linter being more aggressive in local development. 
Switching to production build solves the issue. Alternatively the line `// eslint-disable-next-line vue/return-in-computed-property` can be used to suppress this behaviour, even in local development..

I made a commit to my fork, but i have yet to figure out how to implement this recent commit into the pull request.

Reply:
I do not think suppressing it is a good solution, it is better to return an empty object I mentioned above

Reply:
> I do not think suppressing it is a good solution, it is better to return an empty object I mentioned above

Okidoki :)

I have discovered the documentation for this rule: https://eslint.vuejs.org/rules/return-in-computed-property.html

Reply:
Alright then, i have changed it to return an empty object now :)

Should be all fine now.